int32 cmd

int32 INIT			   = 1
int32 REPOSO 		   = 2
int32 SAFETY 		   = 3
int32 RESET			   = 4
int32 TELEOPERATION    = 5
int32 AUTONOMOUS	   = 6
int32 SEMIAUTONOMOUS   = 7
int32 CHANGE 		   = 8
int32 SUCCEEDED		   = 9
int32 ORIENTED 		   = 10
int32 NOT_ORIENTED     = 11
int32 POSITIONED       = 12
int32 NOT_POSITIONED   = 13
                